core/src/main/java/org/elasticsearch/search/aggregations/AggregatorBuilder.java

```diff
@@ -71,39 +71,16 @@ protected AggregatorBuilder(StreamInput in, Type type) throws IOException {
         metaData = in.readMap();
     }
 
-    protected boolean usesNewStyleSerialization() { // NORELEASE remove this before 5.0.0GA, when all the aggregations have been migrated
```

Comment:
The vast, vast, vast majority of this PR removes all the overrides of this method that I added while migrating the aggregations.